cmd/podman/inspect.go

```diff
@@ -127,6 +127,8 @@ func iterateInput(c *cli.Context, args []string, runtime *libpod.Runtime, inspec
 				break
 			}
 		case inspectTypeImage:
+			newImage := runtime.NewImage(input)
+			newImage.GetLocalImageName()
```

Comment:
Do we do anything with `newImage`? It looks like you never use it besides populating fields in it

Reply:
Should the next line be `GetImage(newImage.LocalName)` as below?

Reply:
I think that is basically what happens in the next line... see email I sent this morning.

Reply:
And yes, primarily to populate fields.

Reply:
I'm still really doubtful about this. It really looks like you should be using `GetImage(newImage.LocalName)` below, or else remove this, because there's no reason to have newImage if you don't use it there

Reply:
k fixed